### PR TITLE
Minor bug fixes and descriptive text for the `LibriCSS` recipe

### PR DESCRIPTION
**Pull request by JinZr**

### CODE REVIEW

File: egs/libricss/SURT/prepare.sh [hunk 1]
```diff
@@ -118,9 +126,12 @@ fi
...
   log "Stage 4: Extract features for LibriSpeech, trim to alignments, and shuffle the cuts"
-  python local/compute_fbank_librispeech.py
-  lhotse combine data/manifests/librispeech_cuts_train* - |\
-    lhotse cut trim-to-alignments --type word --max-pause 0.2 - - |\
+  # python local/compute_fbank_librispeech.py
+  lhotse combine data/manifests/librispeech_cuts_train* data/manifests/librispeech_cuts_train_all.jsonl.gz
+  lhotse cut trim-to-alignments --type word --max-pause 0.2 \
+    data/manifests/librispeech_cuts_train_all.jsonl.gz \
+    data/manifests/librispeech_cuts_train_all_trimmed.jsonl.gz
+  cat <(gunzip -c data/manifests/librispeech_cuts_train_all_trimmed.jsonl.gz) | \
```
Comment (desh2608) [inline egs/libricss/SURT/prepare.sh:134]: I think the original command works fine without having to save all the intermediate manifests?

**Comment by desh2608** [comment]
Let's keep this PR as WIP as you work through the recipe and hopefully fix some more bugs :)

File: egs/libricss/SURT/prepare.sh (hunk 1 above)
Comment (desh2608) [reply egs/libricss/SURT/prepare.sh:134]: @JinZr is this change needed?

Comment (JinZr) [reply egs/libricss/SURT/prepare.sh:134]: Yes i suppose. The old version does not work on my side.

Comment (desh2608) [reply egs/libricss/SURT/prepare.sh:134]: Hmm, weird. Well, anyway, it's a minor issue.

**Pull request merged**